agito.py

```diff
@@ -645,23 +645,50 @@ def substitute(m):
 
 	return message
 
+def use_timezone(timezone):
+    """A git filter-branch filter that converts author and commit times to specifiedtimezone"""
+
+    # import locally, the pytz and dateutil Python modules are not required normally
```

Comment:
the same comment states why the import is local, because the module is not neccessary if not using this "hook"

i find it waste of energy implementing `pytz` over again in this code. timezone handling is not simple thing, mistakes are easy to come.

perhaps try importing and if failed, show some better error message?

something like:
```python
try:
  import pytz
  HAS_PYTZ = True
except ImportError:
  HAS_PYTZ = False
```